plugins/ptp_operator/stats/stats.go

```diff
@@ -236,7 +236,7 @@ func (s *Stats) GetStateState(processName string, iface *string) (ptp.SyncState,
 				return d[0].State, nil
 			}
 			for _, state := range d {
-				if *state.IFace == *iface {
+				if iface != nil && *state.IFace == *iface {
```

Comment:
should check state != nil

Reply:
No required , ranging inputState (state) struct and iface is a pointer .